include/dlaf/gpu/blas/error.h

```diff
@@ -18,67 +18,55 @@
 #include "dlaf/common/source_location.h"
 #include "dlaf/gpu/blas/api.h"
 #include "dlaf/gpu/cublas/error.h"
-#include "dlaf/gpu/hipblas/error.h"
+#include "dlaf/gpu/rocblas/error.h"
 
 namespace dlaf::gpublas::internal {
 
 #ifdef DLAF_WITH_GPU
 
 inline void checkError(gpublasStatus_t st,
                        const dlaf::common::internal::source_location& info) noexcept {
+  if (st != GPUBLAS_STATUS_SUCCESS) {
 #ifdef DLAF_WITH_CUDA
-  if (st != CUBLAS_STATUS_SUCCESS) {
     std::cout << "[CUBLAS ERROR] " << info << std::endl
               << cublas::internal::getErrorString(st) << std::endl;
-    std::terminate();
-  }
 #elif defined(DLAF_WITH_HIP)
-  if (st != HIPBLAS_STATUS_SUCCESS) {
-    std::cout << "[HIPBLAS ERROR] " << info << std::endl
-              << hipblas::internal::getErrorString(st) << std::endl;
+    std::cout << "[ROCBLAS ERROR] " << info << std::endl
+              << rocblas::internal::getErrorString(st) << std::endl;
+#endif
     std::terminate();
   }
-#endif
 }
 
+#define DLAF_GPUBLAS_CHECK_ERROR(gpublas_err) \
+  ::dlaf::gpublas::internal::checkError((gpublas_err), SOURCE_LOCATION())
+
+#endif
+
 #ifdef DLAF_WITH_HIP
-// Lifted from hipBLAS/library/src/hcc_detail/hipblas.cpp, with missing
-// enumeration values added. This should be removed when hipBLAS is no longer
-// used.
-inline hipblasStatus_t rocBLASStatusToHIPStatus(rocblas_status error) {
+inline hipblasStatus_t acceptWorkspaceStatus(rocblas_status error) {
   switch (error) {
     case rocblas_status_continue:
     case rocblas_status_size_unchanged:
     case rocblas_status_size_increased:
     case rocblas_status_success:
-      return HIPBLAS_STATUS_SUCCESS;
+      return rocblas_status_success;
     case rocblas_status_invalid_handle:
-      return HIPBLAS_STATUS_NOT_INITIALIZED;
     case rocblas_status_not_implemented:
-      return HIPBLAS_STATUS_NOT_SUPPORTED;
     case rocblas_status_invalid_pointer:
     case rocblas_status_invalid_size:
     case rocblas_status_invalid_value:
-      return HIPBLAS_STATUS_INVALID_VALUE;
     case rocblas_status_memory_error:
-      return HIPBLAS_STATUS_ALLOC_FAILED;
     case rocblas_status_internal_error:
-      return HIPBLAS_STATUS_INTERNAL_ERROR;
     case rocblas_status_perf_degraded:
     case rocblas_status_size_query_mismatch:
     case rocblas_status_check_numerics_fail:
-    default:
-      return HIPBLAS_STATUS_UNKNOWN;
+      return error;
```

Comment:
I can't add a diff here, but you can use `default` here for the "all-the-other-error-codes" case.